app/src/main/java/com/ej/rovadiahyosefcalendar/classes/SiddurMaker.java

```diff
@@ -930,33 +946,29 @@ public ArrayList<HighlightString> getShacharitPrayers() {
+            "בָּרוּךְ אֱלֹהֵֽינוּ שֶׁבְּרָאָֽנוּ לִכְבוֹדוֹ, וְהִבְדִּילָֽנוּ מִן הַתּוֹעִים, וְנָֽתַן לָֽנוּ תּוֹרַת אֱמֶת, וְחַיֵּי עוֹלָם נָטַע בְּתוֹכֵֽנוּ. הוּא יִפְתַּח לִבֵּֽנוּ בְּתֽוֹרָתוֹ, וְיָשִׂים בְּלִבֵּֽנוּ אַֽהֲבָתוֹ וְיִרְאָתוֹ לַֽעֲשׂוֹת רְצוֹנוֹ, וּלְעָבְדוֹ בְּלֵבָב שָׁלֵם. לֹא נִיגַע לָרִיק, וְלֹא נֵלֵד לַבֶּֽהָלָה. יְהִי רָצוֹן מִלְּפָנֶֽיךָ, יְהֹוָה אֱלֹהֵֽינוּ וֵֽאלֹהֵי אֲבוֹתֵֽינוּ, שֶׁנִּשְׁמוֹר חֻקֶּֽיךָ וּמִצְוֹתֶֽיךָ בָּעוֹלָם הַזֶּה, וְנִזְכֶּה, וְנִחְיֶה, וְנִירַשׁ טוֹבָה וּבְרָכָה לְחַיֵּי הָעוֹלָם הַבָּא: לְמַ֤עַן ׀ יְזַמֶּרְךָ֣ כָ֭בוֹד וְלֹ֣א יִדֹּ֑ם (יפסיק מעט) יְהֹוָ֥ה אֱ֝לֹהַ֗י לְעוֹלָ֥ם אוֹדֶֽךָּ׃ יְהֹוָ֥ה חָפֵ֖ץ לְמַ֣עַן צִדְק֑וֹ יַגְדִּ֥יל תּוֹרָ֖ה וְיַאְדִּֽיר: וְיִבְטְח֣וּ בְ֭ךָ יוֹדְעֵ֣י שְׁמֶ֑ךָ כִּ֤י לֹֽא־עָזַ֖בְתָּ דֹרְשֶׁ֣יךָ יְהֹוָֽה: יְהֹוָ֥ה אֲדֹנֵ֑ינוּ מָה־אַדִּ֥יר שִׁ֝מְךָ֗ בְּכׇל־הָאָֽרֶץ׃ חִ֭זְקוּ וְיַאֲמֵ֣ץ לְבַבְכֶ֑ם כׇּל־הַ֝מְיַחֲלִ֗ים לַֽיהֹוָֽה: \n\n" +
+            "ואומר החזן קדיש תתקבל (And the chazzan says Kaddish Titkabal)"
+        );
         addKaddishTitkabal(false);
 
         if (jewishDateInfo.getJewishCalendar().getDayOfWeek() == 2 || jewishDateInfo.getJewishCalendar().getDayOfWeek() == 5 || jewishDateInfo.getJewishCalendar().isPurim() || jewishDateInfo.getJewishCalendar().isTaanis()) {
             addToSiddur(
                     "ומחזירין את ספר התורה למקומו ואומרים יהללו (Sefer torah is returned)\n\n" +
                             "יְהַלְל֤וּ ׀ אֶת־שֵׁ֬ם יְהֹוָ֗ה כִּֽי־נִשְׂגָּ֣ב שְׁמ֣וֹ לְבַדּ֑וֹ ה֝וֹד֗וֹ עַל־אֶ֥רֶץ וְשָׁמָֽיִם׃ וַיָּ֤רֶם קֶ֨רֶן ׀ לְעַמּ֡וֹ תְּהִלָּ֤ה לְֽכׇל־חֲסִידָ֗יו לִבְנֵ֣י יִ֭שְׂרָאֵל עַ֥ם קְרֹב֗וֹ הַֽלְלוּיָֽהּ׃ יְהֹוָה הוּא הָאֱלֹהִים, יְהֹוָה֙ ה֣וּא הָֽאֱלֹהִ֔ים בַּשָּׁמַ֣יִם מִמַּ֔עַל וְעַל־הָאָ֖רֶץ מִתָּ֑חַת אֵ֖ין עֽוֹד: אֵין־כָּמ֖וֹךָ בָאֱלֹהִ֥ים ׀ אֲדֹנָ֗י וְאֵ֣ין כְּֽמַעֲשֶֽׂיךָ׃ הֲשִׁיבֵ֨נוּ יְהֹוָ֤ה ׀ אֵלֶ֙יךָ֙ וְֽנָשׁ֔וּבָה חַדֵּ֥שׁ יָמֵ֖ינוּ כְּקֶֽדֶם׃"
             );
+        } else if (jewishDateInfo.getJewishCalendar().isTishaBav()) {
```

Comment:
This will not execute because the previous if statement will be true for isTaanis